bigquery/tables.rb

```diff
@@ -23,12 +23,32 @@ def create_table project_id:, dataset_id:, table_id:
   bigquery = Google::Cloud::Bigquery.new project: project_id
   dataset  = bigquery.dataset dataset_id
 
-  dataset.create_table table_id
+  table    = dataset.create_table table_id do |schema|
```

Comment:
Maybe change `|schema|` to `|updater|` or `|table|`?

The block for `create_table` is actually a "Table::Updater" (I see that's not documented in the API docs, so I just created https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2038). It lets you set any property on the table, so more than just add schema fields. \Ffor example, it'll be how you set the encryption key once that feature is released).